apps/prairielearn/src/tests/manualGrading.test.ts

```diff
@@ -375,19 +380,20 @@ describe('Manual Grading', function () {
   before('add staff users', async () => {
     await Promise.all(
       mockStaff.map(async (staff) => {
-        const courseStaffParams = [1, staff.authUid, 'None', 1];
-        const courseStaffResult = await sqldb.callAsync(
-          'course_permissions_insert_by_user_uid',
-          courseStaffParams,
-        );
-        assert.equal(courseStaffResult.rowCount, 1);
-        staff.user_id = courseStaffResult.rows[0].user_id;
-        const ciStaffParams = [1, staff.user_id, 1, 'Student Data Editor', 1];
-        const ciStaffResult = await sqldb.callAsync(
-          'course_instance_permissions_insert',
-          ciStaffParams,
-        );
-        assert.equal(ciStaffResult.rowCount, 1);
+        const courseStaffPermissions = await insertCoursePermissionsByUserUid({
```

Comment:
Nit: name this `user` instead, maybe? AFAICT this is actually a `users` row.

Reply:
Replaced in f95aee4418b67eb0758225db133be1434a03e71d with `const { user_id }`.